lib/internal/module.js

```diff
@@ -44,3 +48,27 @@ function stripBOM(content) {
   }
   return content;
 }
+
+exports.builtinLibs = ['assert', 'buffer', 'child_process', 'cluster',
+  'crypto', 'dgram', 'dns', 'domain', 'events', 'fs', 'http', 'https', 'net',
+  'os', 'path', 'punycode', 'querystring', 'readline', 'stream',
+  'string_decoder', 'tls', 'tty', 'url', 'util', 'v8', 'vm', 'zlib'];
```

Comment:
+1 for having an API that exposes the list. Let's be sure to make it read-only so we don't end up with another `process.config` situation ;-)